owl/EASE-OBJ.owl

```diff
@@ -1876,6 +1889,25 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>The description of an object from functionality point of view.</rdfs:comment>
```

Comment:
Not sure if this is entirely accurate. At least it would need more explanation of what a function of an object is. I think objects can serve a function even though they were not designed for that function. So there could be functional descriptions of objects without any functional design, or?

Reply:
Maybe easiest is to replace *Description* with *Design*